spec/services/hyrax/listeners/file_metadata_listener_spec.rb

```diff
@@ -5,7 +5,7 @@
   let(:data)         { { metadata: metadata, user: user } }
   let(:event)        { Dry::Events::Event.new(event_type, data) }
   let(:fake_adapter) { FakeIndexingAdapter.new }
-  let(:file_set)     { FactoryBot.valkyrie_create(:hyrax_file_set) }
+  let(:file_set)     { FactoryBot.valkyrie_create(:hyrax_file_set, with_index: false) }
```

Comment:
The listener does the indexing and the tests expect the file set isn't already indexed so don't index the file set.